pkg/component/component_test.go

```diff
@@ -29,6 +31,281 @@ import (
 	. "github.com/openshift/odo/pkg/config"
 )
 
+// func TestGomockGetComponentFrom(t *testing.T) {
```

Comment:
I have commented out `TestGomockGetComponentFrom` test because currently facing some issues while retrieving the returned value in gomock framework